app/components/Input.tsx

```diff
@@ -186,20 +170,35 @@ class Input extends React.Component<Props> {
             ))}
           <Outline focused={this.focused} margin={margin}>
             {icon && <IconWrapper>{icon}</IconWrapper>}
-            <InputComponent
-              // @ts-expect-error no idea why this is not working
```

Comment:
Changed to the ternary to remove this.

Did it while debugging to make sure this type issue wasn't my issue, and thought it was worth keeping in the end. Let me know if you have other thoughts!